addons/base_import/static/src/less/base_import.less

```diff
@@ -128,6 +128,11 @@
 
 }
 
+/* Default Color for placeholder on import fields*/
+.o_import_selection .select2-default{
```

Comment:
here, you can remove o_import_selection, and move .select2-default into the oe_import section